File: Project Euler ~ C++/Euler001.cpp
#include <list>
#include <iostream>

using namespace std;

int main() {
	
	list<int> result;
	list<int>::iterator it;
	it = result.begin();

	for(int i; i < 1000;i++){

	if (i % 3 == 0 || i % 5 == 0){
		result.insert(it,i); }
	
	}

	int sum = 0;
	for (auto& n : result){
    	sum += n;
	}

	cout << "The answer is: " << sum << endl;

	return 0;
}

File: Project Euler ~ Rust/Euler001.rs
fn euler1() {
   
   let mut sum = 0;
   
   for x in 1..1000 {
        if x % 3 == 0 || x % 5 == 0 {
        sum += x; }  }
        

    println!("The answer is: {}", sum); }

fn main() {
euler1(); }